Blockzilla/HomeView.swift

```diff
@@ -112,7 +122,7 @@ class HomeView: UIView {
         
         tipTitleLabel.snp.makeConstraints { make in
             make.centerX.equalToSuperview()
-            make.bottom.equalTo(tipDescriptionLabel.snp.top).offset(-UIConstants.layout.homeViewTextOffset)
+            make.bottom.equalTo(tipDescriptionLabel.snp.top).offset(UIConstants.layout.homeViewTextOffset)
```

Comment:
While this constraint is really close to what we want, it's not actually based off of centering it in the view. We should use this constraint like so:

```if textLogo.isHidden {
make.bottom.equalTo(tipDescriptionLabel.snp.top).offset(UIConstants.layout.homeViewTextOffset)
} else {
// Center vertically between the navigation buttons and the URL bar
}

Let me know if you need any help figuring out these constraints!

```

Reply:
The newest screenshots are from the new code I'vent upload the newest code yet

Reply:
Gotcha, thank you for the clarification. Just assign me for review when it's ready for me to look at 😄